crates/driver/Cargo.toml

```diff
@@ -30,6 +34,7 @@ thiserror = "1.0"
 tokio = { version = "1.15", features = ["macros", "rt-multi-thread", "time", "test-util", "signal"] }
 tracing = "0.1"
 warp = { version = "0.3", default-features = false }
+web3 = { version = "0.18", default-features = false }
```

Comment:
Since `ethcontract` re-exports `web3` crate, it makes sense to me to not manage this as a dependency.

No strong preference though.

Reply:
TBH I wasn't even aware that `ethcontract` exports `web3`. From that point of view I think it would make sense to be as explicit about a crates dependencies as possible to avoid confusion.